.github/workflows/test-changed.yml

```diff
@@ -28,3 +28,27 @@ jobs:
       run: yarn build
     - name: Run tests on changed packages
       run: xvfb-run yarn test:changed
+
+  health-metrics-test:
+    name: Health Metrics Test
+    runs-on: ubuntu-latest
+    env:
+      METRICS_SERVICE_URL: ${{ secrets.METRICS_SERVICE_URL }}
+      GITHUB_PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+      GITHUB_PULL_REQUEST_BASE_SHA: ${{ github.event.pull_request.base.sha }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
```

Comment:
Is it for connecting to your service?

Reply:
Yes. The metrics service, the one that receives size measurements, calculates the diff and posts reports back to pull requests as a comment, is hosted on GCP. Google cloud SDK is needed here to authenticate our size test so that it is allowed to call the metrics service.